scripts/components/OpenSearch/build.sh

```diff
@@ -13,6 +13,7 @@ function usage() {
     echo ""
     echo "Arguments:"
     echo -e "-v VERSION\t[Required] OpenSearch version."
+    echo -e "-q QUALIFIER\t[Optional] Version qualifier."
```

Comment:
Do we have any defaults if nothing is specified as a qualifier?

Reply:
Default is not set, which is today. Since it's optional doesn't make sense to say it twice.